From f0b53eb6a43cf17c577c6243575c0f955e274196 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Wed, 30 May 2018 15:33:53 +0800
Subject: [PATCH] arm64: dts: rockchip: remove #sound-dai-cells for RK3308 EVB
 acodec

The RK3308 codec is attached with multi-codec machine
driver, not simple-card, so we don't need use the
property #sound-dai-cells any more.

Change-Id: Ie1afdc6bf2820efbb0054ea5e6d91d9c9025351e
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 4 ----
 arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi | 4 ----
 2 files changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index 8b5e7dfcc8a9..cf8a61b0945c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -239,8 +239,6 @@
 &acodec {
 	status = "okay";
 
-	#sound-dai-cells = <0>;
-
 	rockchip,loopback-grp = <3>;
 	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
@@ -634,8 +632,6 @@
 
 &i2s_8ch_2 {
 	status = "okay";
-
-	#sound-dai-cells = <0>;
 };
 
 &nandc {
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
index 43f89939e713..0c8d9f3ab7d6 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
@@ -245,8 +245,6 @@
 &acodec {
 	status = "okay";
 
-	#sound-dai-cells = <0>;
-
 	rockchip,loopback-grp = <0>;
 	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
@@ -640,8 +638,6 @@
 
 &i2s_8ch_2 {
 	status = "okay";
-
-	#sound-dai-cells = <0>;
 };
 
 &nandc {
-- 
2.35.3

